kml/types.cpp

```diff
@@ -30,6 +30,48 @@ void SetBookmarksMinZoom(FileData & fileData, double countPerTile, int maxZoom)
   minZoomQuadtree.SetMinZoom(countPerTile, maxZoom, setMinZoom);
 }
 
+bool MultiGeometry::IsValid() const
+{
+  return !m_lines.empty();
```

Comment:
nit: `ASSERT_EQUAL(m_lines.size(), m_timestamps.size(), ());` still may be helpful here.